examples/RayleighQuotient.qmd

```diff
@@ -162,7 +162,7 @@ We can also benchmark both
 @benchmark gradient_descent($M, $f, $grad_f, $p0)
 ```
 
-We see, that the conversion costs a bit of performance, but if the Euclidean gradient is easier to compute, this might still be ok.
+We see, that the conversion costs a bit of performance, but if the Euclidean gradient is easier to compute, this might still be OK.
```

Comment:
Why do you want to capitalise OK? I think “ok” is also ok. Maybe it is overall a bit informal so we could maybe rephrase the final part of the sentence anyways.

Reply:
Capitalized OK is more formal: https://www.writing-skills.com/ok-ok-okay-how-do-you-write-ok , so I prefer to use it in technical writing (like here).

Reply:
oKaY :wink:

Reply:
Rephrased it.